Lake/DSL/Script.lean

```diff
@@ -11,17 +11,17 @@ namespace Lake.DSL
 open Lean Parser Command
 
 syntax scriptDeclSpec :=
-  ident (ppSpace "(" Term.simpleBinder ")")? (declValSimple <|> declValDo)
+  ident (ppSpace "(" ident (" : " term)? ")")? (declValSimple <|> declValDo)
```

Comment:
@tydeu Does this part look good now? I don't want to break Lake in today's nightly.

Reply:
Yep, it looks good to me.